wordpress-mobile/WordPress-iOS
# Login Epilogue: adjust table width for iPad

Ref #13413 

On an iPad (specifically, the Regular x Regular size class):
- The table is 50% of the view width.
- The Done button width equals the table width.

For all other size classes, the table and button are as they are currently. That is, constrained to the edges of the view.

To test:
- Login on an iPad.
- Verify the table and button width are 50% of the view width, in portrait and landscape.
- Login on an iPhone, or multitask on an iPad and change the view size.
- Verify the table and button are still constrained to the view edges.

| ![ipad_portrait](https://user-images.githubusercontent.com/1816888/75590637-fe57ee80-5a3a-11ea-84bf-4a822dfa4d66.jpeg) | ![ipad_multitasking_portrait](https://user-images.githubusercontent.com/1816888/75590656-06b02980-5a3b-11ea-8a83-6dbf40d660e9.jpeg)  |
|--------|-------|
| ![ipad_landscape](https://user-images.githubusercontent.com/1816888/75590670-129beb80-5a3b-11ea-9fe6-e6e63dbc7451.jpeg) | ![ipad_multitasking_landscape](https://user-images.githubusercontent.com/1816888/75590678-1af42680-5a3b-11ea-8e35-19c62bb35474.jpeg)  |

PR submission checklist:
